File: Codigo_trabalho/Rodas.cpp
#include "Rodas.h"
#include <iostream>
#include <cstdlib>

using namespace std;

Rodas::Rodas()
{
    calibragem_pneu = rand() % 2;
}

void Rodas::set_calibragem(bool c_p)
{
    calibragem_pneu = c_p;
}

bool Rodas::get_calibragem()
{
    return calibragem_pneu;
}

void Rodas::imprimir()
{
    cout << "\n Calibragem do Pneu: " << calibragem_pneu << endl;
}

File: POO/C++/Trabalho_POO_parte_2/Rodas.py
import random

class Rodas:
    def __init__(self):
        self.calibragem_pneu = random.randint(0, 1)

    def set_calibragem(self, c_p):
        self.calibragem_pneu = c_p

    def get_calibragem(self):
        return self.calibragem_pneu

    def imprimir(self):
        print(f"\nCalibragem do Pneu: {self.calibragem_pneu}")
